install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -65,12 +65,12 @@ spec:
           - --controlPlaneAuthPolicy
           - MUTUAL_TLS
           - --discoveryAddress
-          - istio-pilot.istio-system:15005
```

Comment:
You might want to parameterize this. Because the gateway can be deployed in a namespace other than istio-system. In that case, the pilot address will not resolve

Reply:
Ok, i understand this one, i've seen the PR that add the hardcoded the istio-system namespace

Reply:
For consistency please customize the full pilot address ( we already have an annotation for the sidecar injector, statsd also customizes the host). Not only namespace - it may be a random host (useful for multi-cluster).